assets/RNAseq_report.Rmd

```diff
@@ -1065,14 +1062,24 @@ sampleDists <- dist(t(assay(if (run_rlog) rld else vsd)))
 sampleDistMatrix <- as.matrix(sampleDists)
 colors = colorRampPalette(rev(RColorBrewer::brewer.pal(9, "Blues")))(255)
 par(oma=c(3,3,3,3))
+dist_heatmap <- pheatmap(mat = sampleDistMatrix, color=colors)
+
+svg("differential_gene_expression/plots/Heatmaps_of_distances.svg")
+dist_heatmap
+dev.off()
```

Comment:
Can you only save it now by calling the heatmap and then closing the graphics device?

Reply:
Well yes, I did not find a pheatmap internal saving function; on the internet code snippets simply use the R graphics functions svg() etc.